keanu-project/src/test/java/io/improbable/keanu/vertices/utility/PrintVertexTest.java

```diff
@@ -0,0 +1,85 @@
+package io.improbable.keanu.vertices.utility;
+
+import com.google.common.collect.ImmutableSet;
+import io.improbable.keanu.algorithms.mcmc.MetropolisHastings;
+import io.improbable.keanu.network.BayesianNetwork;
+import io.improbable.keanu.vertices.dbl.DoubleVertex;
+import io.improbable.keanu.vertices.dbl.nonprobabilistic.ConstantDoubleVertex;
+import io.improbable.keanu.vertices.dbl.probabilistic.UniformVertex;
+import java.io.PrintStream;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.atLeast;
+import static org.mockito.Mockito.verify;
+
+public class PrintVertexTest {
+    @Mock
+    PrintStream printStream;
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void testWhenCreatedThenParentIsSet() {
```

Comment:
This might be in our codebase in some places but I like to avoid prepending every test name with “test”. I used to try to name them using the Given When Then approach but that’s not always practical in camelCase. https://martinfowler.com/bliki/GivenWhenThen.html

Reply:
👍